draft-ietf-trans-rfc6962-bis.md

```diff
@@ -195,8 +196,9 @@ such mechanisms are outside the scope of this document.
 ## Requirements Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in [RFC2119].
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
```

Comment:
@eranmes I'm afraid I couldn't figure out how to express the RFC8174 boilerplate without resorting to using implicit references.  (I know we'd been avoiding using implicit references in this document, but I honestly don't remember why ;-) ).

Reply:
Not sure either. LGTM (but I'll point out we  don't use the 'NOT RECOMMENDED' term in the document)

Reply:
Thanks Eran.  I agree that we don't use "NOT RECOMMENDED".  However, Alissa asked us to "use the RFC 8174 boilerplate", which I presume means "copy and paste the RFC 8174 boilerplate, without editing it".